### PR TITLE
Create SECURITY.md

### LINKED ISSUE #33385: Add a security policy for reporting issues
Set up a security policy.

Help the community understand how to securely report security vulnerabilities for calypso.

https://github.com/Automattic/wp-calypso/security/policy

### PR DESCRIPTION
**Pull request by sarayourfriend**
#### Changes proposed in this Pull Request

* Add a security policy

Note: I just took a stab at copying Jetpack's policy from here: https://raw.githubusercontent.com/Automattic/jetpack/master/SECURITY.md

I *think* we want to add a blurb about WP Desktop, so I've left a section open for that. @nsakaimbo could you help with that part?

Fixes #33385

### CODE REVIEW

**Comment by nsakaimbo** [comment]
Thanks for the ping, @saramarcondes! I'm not too well-versed here, but my educated guess is we'd probably want to defer to Automattic's security policy as well. @belcherj Any thoughts here? Or perhaps we can borrow from the language used by Simplenote?

Quick question:

It says Automattic's Security Policy is located at [https://automattic.com/security/](https://automattic.com/security/) - however that link doesn't appear to be a policy? (This page only states resources for support and to report vulnerabilities - not exactly policy documentation?)

File: SECURITY.md
Comment (tyxla) [inline SECURITY.md:23]: Should we also include Jetpack cloud here?

```suggestion
* wordpress.com
* cloud.jetpack.com
```

File: SECURITY.md
Comment (tyxla) [inline SECURITY.md]: I'm not completely sure what this means. Calypso doesn't really have versions, so can we make this more accurate somehow?

File: SECURITY.md
Comment (tyxla) [inline SECURITY.md]: I'm assuming this will replicate what we've currently written above for Web, but I'll defer to @nsakaimbo to confirm.

**Review by tyxla — COMMENTED** [review]
Thanks for this one 👍 

Looks good to me in general. Left some thoughts.

File: SECURITY.md
Comment (sarayourfriend) [reply SECURITY.md]: I wasn't sure about this either... What's the best way to say we only support the currently live version of Calypso?

Comment (tyxla) [reply SECURITY.md]: Maybe we could reference the `latest version in trunk` here? That would make sense. And we could further clarify it with something like "Calypso is continuously deployed shortly after each merge, multiple times per day."

**Comment by sarayourfriend** [comment]
@nsakaimbo I've updated the section about WordPress Desktop. Please let me know if it makes sense/is correct.

@tyxla I've updated the document generally by copying the blurb about security from the `automattic.com/security` page rather than linking to it. I think this makes sense because the relevant aspects of that page (application security) are difficult to immediately locate on the page given the prominence of account security details on it.

File: SECURITY.md
Comment (sarayourfriend) [reply SECURITY.md]: Done.

File: SECURITY.md
Comment (tyxla) [inline SECURITY.md]: Re-reading it makes me think that a bit of specificity would help here:

```suggestion
Only the latest version of Calypso (that is, what is present in the `trunk` branch) is supported. Calypso is continuously deployed shortly after each merge to trunk, multiple times per day.
```

**Review by tyxla — APPROVED** [review]
Not much to add 👍  LGTM from my perspective

**Comment by sarayourfriend** [comment]
Merging, @nsakaimbo if you end up needing to make changes or need help with this just ping us 🙂

**Pull request merged**
